Sources/Instructions/Managers/Internal/OverlayStyleManager/TranslucentOverlayStyleManager.swift

```diff
@@ -10,17 +10,20 @@ class TranslucentOverlayStyleManager: OverlayStyleManager {
     // MARK: Private Properties
     private var onGoingTransition = false
     private let color: UIColor
-
+    private var cutoutBorderColor: UIColor
+    
     // MARK: Layer Mask related properties
     private var cutoutMaskLayer = CAShapeLayer()
     private var fullMaskLayer = CAShapeLayer()
+    private var borderLayer = CAShapeLayer()
     private lazy var overlayLayer: CALayer = {
         return self.createSublayer()
     }()
 
     // MARK: Initialization
-    init(color: UIColor) {
+    init(color: UIColor, cutoutBorderColor: UIColor) {
```

Comment:
I believe that providing `.clear` as the default value of `cutoutBorderColor` would prevent tests from breaking.